collectors/ebpf.plugin/README.md

```diff
@@ -191,6 +191,53 @@ The eBPF collector enables and runs the following eBPF programs by default:
 -   `network viewer`: This eBPF program creates charts with information about `TCP` and `UDP` functions, including the
     bandwidth consumed by each.
 
+### `[network viewer]`
+You can configure the information shown on `outbund` and `inbound` charts with the settings in this section. 
+
+```conf
+[network viewer]
+    maximum dimensions = 500
+    included ports = 19999 100-1024
+    excluded ports = 145 139
+    included services = ssh http
+    excluded services = ftp
+    included hostnames = netdata.cloud netdata.io
+    excluded hostnames = example.com
+    excluded ips = 127.0.0.1 192.168.0.1-192.168.0.5
+    included ips = 192.168.0.0/24 0001:0000:0000:0000:0000:0000:0000:0000/32 
+```
+
+When you define an `included` setting, Netdat will collect network metrics for that specific port. For example, if you
```

Comment:
Netdat->Netdata,
port->port/service/hostname/ip?

Reply:
In general, even when I approve please make sure @joelhans takes a look at it before merging. There are multiple errors/strange formulations in the documentation. But I am not a native English speaker as well.

Reply:
I observed that when I copied and I pasted the format was cut, but the majority of the original text given by @joelhans  was kept. 
I fixed the typo that you requested. 
Now about the transition from `port` to `port/service/hostname/ip`, this would force to rewrite the whole paragraph. I understand that is enough to give an example for a pair of `included` and `excluded`, because all pairs works the same way.

Reply:
> Now about the transition from port to port/service/hostname/ip...

Then you should write
```
When you define the `included ports` setting
```
And mention that other sections have similar behavior.

Reply:
I talked with @joelhans and we used @vlvkobal suggestion to update the paragraph. Thank you guys!

Reply:
> 0001:0000:0000:0000:0000:0000:0000:0000/32

> https://en.wikipedia.org/wiki/IPv6_address#Representation

=> `1::/32`

Does your parser handle all cases?

> https://tools.ietf.org/html/rfc5952

Where did you get the ip address parser? Is it your custom? Are there any tests?

Reply:
> Does your parser handle all cases?

Yes, @ilyam8 the parser is done using `inet_pton` to do the conversion from string to network byte order, so all the values present in the RFC should be parsed in this function. 

> Where did you get the ip address parser?

The function `inet_pton` is defined inside `<arpa/inet.h>` and it is part of POSIX, you can see more details in this [link](https://pubs.opengroup.org/onlinepubs/9699919799/functions/inet_ntop.html)

>  Is it your custom? 

No, I never considered this possibility, because we would need to cover different formats for IPV4 and IPV6, so I decided to use the Linux functions.

> Are there any tests?

I compared the function output with IPV6 calculators around the internet, I also used `inet_ntop` to confirm that values converted from string to binary could be reversed from binary to string.

Reply:
ok, thx

anyway `0001:0000:0000:0000:0000:0000:0000:0000/32` looks weird, nobody writes it like that

i suggest to use [`2001:DB8::/32`](https://tools.ietf.org/html/rfc3849) or whatever you like but written in _correct_ representation

Reply:
Thank you for the suggestion, I updated the documentation using the format you suggested.